config.dev.yaml

```diff
@@ -0,0 +1,35 @@
+issuer: http://127.0.0.1:5556/dex
+
+storage:
+  type: memory
```

Comment:
```suggestion
  type: sqlite3
```

Reply:
I changed it to memory on purpose. This is supposed to be a dummy config. Without the proper volume mapping changing it to sqlite works the same way.